forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -148,7 +216,28 @@ class ForageTerminalSDK(
     override suspend fun capturePayment(
         params: CapturePaymentParams
     ): ForageApiResponse<String> {
-        return forageSdk.capturePayment(params)
+        val logger = createLogger()
+        val (_, paymentRef) = params
+
+        logger.i(
+            "[POS] Called capturePayment for Payment $paymentRef",
+            attributes = mapOf(
+                "payment_method_ref" to paymentRef
+            )
+        )
+
+        val captureResponse = forageSdk.capturePayment(params)
```

Comment:
Hmmmm we are going to get some extra log statements I think by calling the original function. It's not the biggest deal, but I would definitely consider it tech debt since we can expect a MASSIVE amount of volume some day with this.